start-here/js/script-main.js

```diff
@@ -18,10 +18,70 @@ function handleSearch() {
     input.classList.add("search-box--warning");
   } else {
     warningMessage.style.display = "none";
+    fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${input.value}`)
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.title) {
+          document.getElementById(
+            "title-wrapper"
+          ).innerHTML = `<p>No definition found for <strong>${input.value}</strong>.</p>`;
+        } else {
+          const meanings = data[0].meanings;
+          const phonetics = data[0].phonetics;
+
+          document.querySelector(".title").innerText = input.value;
```

Comment:
Line 32 could be better to use `data[0].word` instead of `input.value`.
It works the same way for this project, but if we decide to extract the fetch function to a separate file, the input.value will not be more accessible but the data is related to the scope of the function.